docs/data/date-pickers/adapters-locale/adapters-locale.md

```diff
@@ -369,3 +369,12 @@ moment.updateLocale('en', {
   },
 });
 ```
+
+## RTL Support
+
+Right-to-left languages such as Arabic, Persian, or Hebrew are supported.
+Follow [this guide](/material-ui/customization/right-to-left/) to use them.
```

Comment:
```suggestion
Follow [this guide](https://mui.com/material-ui/customization/right-to-left/) to use them.
```

Reply:
Why should we make this URL absolute?
It will be deployed in the same domain

Reply:
My bad, I didn't notice this was on material-ui docs 🙈